### PR TITLE
Bugfix/general 23

### PR DESCRIPTION
**Pull request by ue**

### CODE REVIEW

File: src/components/notification/view/notificationView.js [hunk 1]
```diff
@@ -60,15 +61,15 @@ class NotificationView extends PureComponent {
     return (
       <FlatList
         data={data}
+        initialNumToRender={data && data.length}
+        maxToRenderPerBatch={data && data.length}
```
Comment (mistikk) [inline src/components/notification/view/notificationView.js]: we can use lodash get

File: src/components/notification/view/notificationView.js [hunk 2]
```diff
@@ -177,47 +172,39 @@ class NotificationView extends PureComponent {
...
-              />
-            )}
-            renderItem={({ item, index }) => (
-              <Fragment>
-                <ContainerHeader
-                  hasSeperator={index !== 0}
-                  isBoldTitle
-                  title={item.title}
-                  key={item.title}
+        { _notifications.length === 0 && true
```
Comment (mistikk) [inline src/components/notification/view/notificationView.js]: if true?

File: src/components/postView/view/postDisplayView.js [hunk 3]
```diff
@@ -86,27 +88,38 @@ class PostDisplayView extends PureComponent {
...
+                    name="delete-forever"
+                    onPress={() => this.ActionSheet.show()}
+                    style={styles.barIconButton}
+                  />
+                )}
+                <IconButton
+                  iconStyle={styles.barIconRight}
+                  iconType="MaterialIcons"
+                  name="create"
+                  onPress={() => handleOnEditPress && handleOnEditPress()}
```
Comment (mistikk) [inline src/components/postView/view/postDisplayView.js]: I will explain with dm

File: src/components/notification/view/notificationView.js (hunk 1 above)
Comment (ue) [reply src/components/notification/view/notificationView.js]: no

File: src/components/notification/view/notificationView.js (hunk 2 above)
Comment (ue) [reply src/components/notification/view/notificationView.js]: @mistikk updated

**Pull request merged**